markdown/Cheapskate/ParserCombinators.hs

```diff
@@ -107,9 +109,14 @@ instance Alternative Parser where
   {-# INLINE empty #-}
   {-# INLINE (<|>) #-}
 
+instance Fail.MonadFail Parser where
+  fail e = Parser $ \st -> Left $ ParseError (position st) e
+
 instance Monad Parser where
   return x = Parser $ \st -> Right (st, x)
-  fail e = Parser $ \st -> Left $ ParseError (position st) e
+#if !MIN_VERSION_base(4,13,0)
+  fail = Fail.fail
+#endif
```

Comment:
this is related to monad fail change in base - this is analogue to elm-format PR changes